assets/airports/kbos.json

```diff
@@ -562,152 +529,6994 @@
             "origin": "kbos",
```

Comment:
From [the airport master record](http://www.gcr1.com/5010ReportRouter/BOS.pdf), the 2016 stats indicate 387,929 aircraft movements. IAW the [traffic level specs](https://github.com/openscope/openscope/blob/develop/documentation/airport-file-standards.md), this should fall between 44acph and 133acph. Currently, there's a total of 61acph.

This is good to go in that regard. :+1: